devel/ansible/roles/datagrepper/tasks/main.yml

```diff
@@ -0,0 +1,120 @@
+---
+- name: Install RPM packages
+  dnf:
+      name:
+      - fedora-messaging
+      - gcc
+      - git
+      - poetry
+      - python3-devel
+      - python3-pip
+      - vim
+      - krb5-devel
+      - libpq-devel
+      - tox
+      - python-psycopg2
+      state: present
+
+- name: Install the .bashrc
+  copy:
+    src: .bashrc
+    dest: /home/vagrant/.bashrc
+    mode: 0644
+    owner: vagrant
+    group: vagrant
+
+- name: check out datanommer from git
+  git:
+    repo: https://github.com/fedora-infra/datanommer.git
+    dest: /home/vagrant/datanommer
+  become: yes
+  become_user: vagrant
+
+- name: Create a directory for the virtualenv
+  file:
+    name: /srv/venv
+    state: directory
+    mode: 0755
+    owner: vagrant
+    group: vagrant
+
+- name: Create the virtualenv and install poetry
+  pip:
+    name: poetry
+    virtualenv: /srv/venv
+  become: yes
+  become_user: vagrant
+
+- name: Install datagrepper with poetry
+  shell: /srv/venv/bin/poetry install
+  args:
+    chdir: /home/vagrant/datagrepper/
+  become: yes
+  become_user: vagrant
+
+- name: Install datanommer.models with poetry
+  shell: /srv/venv/bin/poetry install
+  args:
+    chdir: /home/vagrant/datanommer/datanommer.models/
+  become: yes
+  become_user: vagrant
+
+- name: Install datanommer.commands with poetry
+  shell: /srv/venv/bin/poetry install
+  args:
+    chdir: /home/vagrant/datanommer/datanommer.commands/
+  become: yes
+  become_user: vagrant
+
+- name: Install datanommer.consumer with poetry
+  shell: /srv/venv/bin/poetry install
+  args:
+    chdir: /home/vagrant/datanommer/datanommer.consumer/
+  become: yes
+  become_user: vagrant
+
+- name: Use the stage fedora-messaging queue to consume
+  copy:
+    remote_src: True
+    src: /etc/fedora-messaging/fedora.stg.toml
+    dest: /etc/fedora-messaging/config.toml
```

Comment:
You also need to change the default queue name from `00000000-0000-0000-0000-000000000000` to a random number generated with `uuidgen`. It's annoying, but otherwise we'll get conflicts.

Reply:
done FWIW